src/app/mi-details/add-problem-statement/add-problem-statement.page.html

```diff
@@ -0,0 +1,53 @@
+<app-application-header [config]="headerConfig"></app-application-header>
+<ion-content class="ion-padding">
+  <div class="container">
+    <h1 class="title">{{"TAG_IMPROVEMENT" | translate}}</h1>
+    <div>
+        <div class="input-container" *ngIf="showInput; else addNewProblemStatement">
```

Comment:
use new if else syntax.. the question was what's the difference between two code blocks